contract-map.json

```diff
@@ -1,4 +1,32 @@
 {
+  "0xc20C9C13E853fc64d054b73fF21d3636B2d97eaB": {
+    "name": "CoW Protocol Virtual Token",
+    "logo": "vCOW.png",
+    "erc20": true,
+    "symbol": "vCOW",
+    "decimals": 18
+  },
+  "0xD057B63f5E69CF1B929b356b579Cba08D7688048": {
+    "name": "CoW Protocol Virtual Token",
+    "logo": "vCOW.png",
```

Comment:
SVG is required

Reply:
Is it? As I saw other token assets also using PNG. If it's a must we can create one.